<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumCreator</class>
 <widget class="QMainWindow" name="AlbumCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>546</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Создание альбома - Blackland Radio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_song_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавить песню</string>
    </property>
   </widget>
   <widget class="QLabel" name="songs_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>29</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Песни</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLabel" name="year_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Год</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="year_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="genre_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genres_combobox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_cover_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбрать обложку</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="photo_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cover">
       <property name="text">
        <string>*photo*</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="create_album_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать альбом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="songs_scroll">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>70</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="songs_contents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>357</width>
       <height>397</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="songs_layout"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
